R/prophet.R

```diff
@@ -239,8 +239,12 @@ Prophet <- function(jaspResults, dataset = NULL, options) {
     prophetModelFullResults  <- .prophetModelResultsHelper(dataset, options)
     prophetResults[["prophetModelResults"]] <- .prophetModelResultsReduce(prophetModelFullResults, options)
 
+    if ((options[["predictionType"]] == "periodicalPrediction" && options[["periodicalPredictionNumber"]] > 0) ||
```

Comment:
This is ok, but perhaps it's better to constrain `periodicalPredictionNumber` directly in QML here:

https://github.com/jasp-stats/jaspProphet/blob/227c5cf3669f97d2a96d083534dde7aa5017b4b3/inst/qml/Prophet.qml#L366

by adding `inclusive: JASP.MaxOnly` to this component

Reply:
I wanted to allow the user to model the data without having to make predictions by being able to set the option to 0. For example if they don't have future covariates to make predictions and just want to take a look at the seasonality of their data.

Reply:
ok!